charts/prometheus/templates/headless-svc.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     - name: http
       port: {{ .Values.server.statefulSet.headless.servicePort }}
       protocol: TCP
-      targetPort: 9090
+      targetPort: {{ default (include "prometheus.port" .) .Values.server.portName }}
```

Comment:
@zeritti I wonder if its worth to set this as another param in helper to not repeat it

Reply:
Thanks, @DrFaust92. I think we can leave it.